CHANGELOG.md

```diff
@@ -1,6 +1,14 @@
 Change Log
 ==========
 
+Version 1.4.3 *(2023-05-30)*
```

Comment:
Thanks for updating the changelog!

Reply:
Just following the steps in RELEASING.md :)